docs/maintenance/operations/docker-images.md

```diff
@@ -32,7 +32,7 @@ it is easier to configure alerts, notifications or scan reports and also GitHub
 
 ## Retention
 
-This section Work In Progress: the development of the Docker images building process is not yet completed.
+This section is a Work In Progress: the development of the Docker images building process is not yet completed.
```

Comment:
People on discord have talked about documenting docker builds more. That could go here since it's clearly a WIP at the moment.